templates/reconciliation.html.erb

```diff
@@ -0,0 +1,53 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" integrity="sha512-dTfge/zgoMYpP7QbHy4gWMEGsbsdZeCXz7irItjcC3sPUFtf0kuFbDz/ixG7ArTxmDjLXDmezHubeNikyKGVyQ==" crossorigin="anonymous">
+    <link rel="stylesheet" rel="http://code.jquery.com/ui/1.11.4/themes/ui-darkness/jquery-ui.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css" rel="stylesheet" />
+    <style>
+.remove { float: right; }
+    </style>
+    <script src="http://code.jquery.com/jquery.js"></script>
+    <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/js/select2.min.js"></script>
+    <script>
+window.headers = <%= Yajl::Encoder.encode(headers, html_safe: true) %>;
+window.matches = <%= Yajl::Encoder.encode(matched, html_safe: true) %>;
+    </script>
+    <script>
+<%= reconciliation_js %>
+    </script>
+
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <h1 class="page-title">Reconciliation</h1>
+          <p>Check that the names on the left match the names on the right. If they don't then press the <code>x</code> to the right of the row and it will drop to the bottom. If the row matches someone in the existing data you can choose their name from the dropdown one the row appears at the bottom of the table.</p>
+          <table class="table">
+            <tr>
+              <th>Existing</th>
+              <th>Incoming</th>
+            </tr>
+          </table>
+          <select class="js-merged-rows" style="display: none;">
+            <option value="">Select an existing person to match with</option>
+            <% merged_rows.sort_by { |r| r[:name] }.each do |row| %>
```

Comment:
Shouldn't this do a uniq-by-UUID filter first (or is that already happening somewhere else that I can't see?)